reflex/event.py

```diff
@@ -115,6 +117,21 @@ class Config:
         # Needed to allow serialization of Callable.
         frozen = True
 
+    @classmethod
+    def __class_getitem__(cls, type_: str) -> _GenericAlias:
```

Comment:
```suggestion
    def __class_getitem__(cls, args_spec: str) -> _GenericAlias:
```

i'd probably call this `args_spec` here for consistency